<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Parachute Encoder</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="titleLabel">
      <property name="styleSheet">
       <string>font-size: 18px; font-weight: bold;</string>
      </property>
      <property name="text">
       <string>Parachute Encoder</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="messageBox">
      <property name="title">
       <string>Message</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="labelMessage">
         <property name="text">
          <string>Your message:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="messageInput"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="viewTabs">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="parachuteViewTab">
       <attribute name="title">
        <string>Parachute View</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="binaryViewTab">
       <attribute name="title">
        <string>Binary View</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3"/>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="controlsBox">
      <property name="title">
       <string>Controls</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="labelSectors">
         <property name="text">
          <string>Sectors:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="sliderSectors">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinSectors"/>
       </item>
       <item>
        <widget class="QLabel" name="labelTracks">
         <property name="text">
          <string>Tracks:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="sliderTracks">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinTracks"/>
       </item>
       <item>
        <widget class="QPushButton" name="colorButton">
         <property name="text">
          <string>Choose Background Color</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="exportButton">
         <property name="text">
          <string>Export Parachute Image</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="fileMenu">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="saveAction"/>
    <addaction name="openAction"/>
    <addaction name="exitAction"/>
   </widget>
   <widget class="QMenu" name="languageMenu">
    <property name="title">
     <string>Change Language</string>
    </property>
    <addaction name="actionEnglish"/>
    <addaction name="actionfrensh"/>
    <addaction name="actionArabic"/>
   </widget>
   <addaction name="languageMenu"/>
   <addaction name="fileMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="saveAction">
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="openAction">
   <property name="text">
    <string>&amp;Open</string>
   </property>
  </action>
  <action name="exitAction">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="actionEnglish">
   <property name="text">
    <string>English</string>
   </property>
  </action>
  <action name="actionfrensh">
   <property name="text">
    <string>French</string>
   </property>
  </action>
  <action name="actionArabic">
   <property name="text">
    <string>Arabic</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>